incubator/cassandra/values.yaml

```diff
@@ -0,0 +1,44 @@
+## Cassandra image version
+## ref: https://hub.docker.com/r/library/cassandra/
+image:
+  repo: "cassandra"
```

Comment:
This image is huge. I am pushing a new image into the cassandra example in kubernetes.  Do we want to use that image?